### PR TITLE
Add coverage for backend/frontend

### PR DESCRIPTION
**Pull request by sambasivaraop**
Add coverage for backend/frontend.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,7 +45,7 @@ nosetests.xml
 coverage.xml
 *.cover
 .hypothesis/
-
+cover/
```
Comment (inovizz) [inline .gitignore]: Can you add a comment here, so that a reviewer can be aware of why this line was added?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # CodeXpose
-[![Build Status](https://travis-ci.org/inovizz/codeXpose.svg?branch=master)](https://travis-ci.org/inovizz/codeXpose)</br>
+[![Build Status](https://travis-ci.org/inovizz/codeXpose.svg?branch=master)](https://travis-ci.org/inovizz/codeXpose)
```
Comment (inovizz) [inline README.md]: The repo should point to imaginea now, not to inovizz.

File: .travis.yml [hunk 3]
```diff
@@ -4,13 +4,32 @@ language: python
...
 script:
   - cd backend/codexpose/
   - pip install -r requirements.txt
   - pylint --rcfile=../../.pylintrc codexpose
   - pycodestyle codexpose/
   - pylint --rcfile=../../.pylintrc interview/
   - pycodestyle interview/
+  - touch codexpose.log
+  - python manage.py test
+#  - nosetests -v --with-coverage --cover-tests --cover-package=interview interview/tests/test*.py
```
Comment (inovizz) [inline .travis.yml]: Is this line not needed? if yes then remove it.

Also, are we generating code coverage as well in Travis?

File: backend/codexpose/codexpose/settings.py [hunk 4]
```diff
@@ -84,7 +91,7 @@
         'TEST_NAME': 'test_codexpose',
         'USER': 'dev',
         'PASSWORD': 'abc@1234',
-        'HOST': 'db',
+        'HOST': '127.0.0.1',
```
Comment (inovizz) [inline backend/codexpose/codexpose/settings.py]: This shouldn't have changed if we plan to use Docker. With 127.0.0.1 docker setup won't work.

So, for now, create an issue for this and update README that docker setup is not working.

File: backend/codexpose/interview/tests/test_models.py [hunk 5]
```diff
@@ -12,8 +11,7 @@ class CodexposeModelTestCase(TestCase):
     """The class contains test cases for hrbot"""
 
     def setUp(self):
-        self.path = os.path.abspath("../../../files/test.txt")
-
+        self.path = "interview/tests/test.txt"
```
Comment (inovizz) [inline backend/codexpose/interview/tests/test_models.py]: Don't use constant values in code, define it somewhere else and use the CONSTANT variable.

File: .gitignore (hunk 1 above)
Comment (sambasivaraop) [reply .gitignore]: Done. Added comment.

File: README.md (hunk 2 above)
Comment (sambasivaraop) [reply README.md]: Changes done for pointing repo to Imaginea

File: .travis.yml (hunk 3 above)
Comment (sambasivaraop) [reply .travis.yml]: Commented line not required. Hence removed.
Yes, we are generating code coverage in Travis as well.

File: backend/codexpose/codexpose/settings.py (hunk 4 above)
Comment (sambasivaraop) [reply backend/codexpose/codexpose/settings.py]: Issue#56 has been created for this host configuration and updated README accordingly.

File: backend/codexpose/interview/tests/test_models.py (hunk 5 above)
Comment (sambasivaraop) [reply backend/codexpose/interview/tests/test_models.py]: Changes done. CONSTANT variable has been used instead of value.

**Pull request merged**